test/fixtures/quoted-sting-match.less

```diff
@@ -0,0 +1,24 @@
+.@{var}-handle:after {
+	content: "Content At quoted String comments";
+	content: "";
+}
+
```

Comment:
Something bothers me about that file :D Why it has duplicate stuff? At least we can make it with different classes and different comments.

Reply:
I just wanted to replicate multiple scenarios. Will modify :)